### PR TITLE
Added quick guide to visualization.

### PR DESCRIPTION
**Pull request by ghost**
The guide shows how to add rack on data-center view and descibes in a
few words both data-center view and rack-view from `DC View`.

### CODE REVIEW

File: doc/quickstart.rst [hunk 1]
```diff
@@ -276,9 +276,45 @@ After generating a report, you can close it by clicking "Accept costs"
...
+
+Adding rack
+-----------
+
+You can also add rack on this view (this is the recommended way of racks
+adding). To achieve that, execute these
+steps:
+
+    * from the main menu (top of Ralph's page) click `DC View`
+    * then pick data center from left side (e.g.: 'DC A')
```
Comment (mkurek) [inline doc/quickstart.rst]: on left side

File: doc/quickstart.rst [hunk 2]
```diff
@@ -276,9 +276,45 @@ After generating a report, you can close it by clicking "Accept costs"
...
+    * click `Edit` tab
+    * put a rack on `data center` map by left click, then you should have image similar to
+    .. image:: _static/visualization_adding_rack.png
+        :scale: 30%
+    * then fill fields `name` and `description` for newly creating rack
```
Comment (mkurek) [inline doc/quickstart.rst:311]: not sure, but shouldn't it be `newly created`?

Comment (ghost) [inline doc/quickstart.rst:311]: I'm not english export, but i can say what i mean..
I used `creating` because the rack is not created yet.

**Comment by ar4s** [comment]
:ok:

**Comment by vi4m** [comment]
LGTM

File: doc/quickstart.rst (hunk 2 above)
Comment (vi4m) [inline doc/quickstart.rst:311]: Change it to '`newly crated` even if it's not exactly the truth. `creating` is even more misleading.

**Pull request merged**